### PR TITLE
Update ve.m3u

### PR DESCRIPTION
**Pull request by sguinetti**
This PR add Italianissimo RCR, and edit some streams

### CODE REVIEW

File: channels/ve.m3u [hunk 1]
```diff
@@ -35,8 +35,12 @@ https://d2fxrfbiedz1tm.cloudfront.net/livepaxtv/smil:PC.smil/playlist.m3u8
 http://vcp1.myplaytv.com/valetv/valetv/playlist.m3u8
 #EXTINF:-1 tvg-id="VePlus.ve" tvg-country="VE" tvg-language="Spanish" tvg-logo="" group-title="",Ve Plus (480p) [Not 24/7]
 http://190.122.96.187:8888/http/006
-#EXTINF:-1 tvg-id="Venevision.ve" tvg-country="VE" tvg-language="Spanish" tvg-logo="https://www.directv.com/images/logos/channels/dark/large/219.png" group-title="",Venevision [Offline]
-https://linkastream.co/headless?url=https://www.youtube.com/watch?v=YNNR0vDH6vA
+#EXTINF:-1 tvg-id="Venevision.ve" tvg-country="VE" tvg-language="Spanish" tvg-logo="https://www.directv.com/images/logos/channels/dark/large/219.png" group-title="",Venevision
+https://linkastream.co/headless?url=https://www.youtube.com/user/venevisionweb/live
```
Comment (LaneSh4d0w) [inline channels/ve.m3u:39]:
```suggestion
http://free.fullspeed.tv/headless?url=https://www.youtube.com/user/venevisionweb/live
```

File: channels/ve.m3u [hunk 2]
```diff
@@ -35,8 +35,12 @@ https://d2fxrfbiedz1tm.cloudfront.net/livepaxtv/smil:PC.smil/playlist.m3u8
...
+#EXTINF:-1 tvg-id="RCR750.ve" tvg-country="VE" tvg-language="Spanish" tvg-logo="" group-title="",Radio Caracas Radio 750
+https://linkastream.co/headless?url=https://www.youtube.com/channel/UC5AA3XP4_pXIELctSsH_L7w/live
```
Comment (LaneSh4d0w) [inline channels/ve.m3u:41]:
```suggestion
http://free.fullspeed.tv/headless?url=https://www.youtube.com/channel/UC5AA3XP4_pXIELctSsH_L7w/live
```

File: channels/ve.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/ve.m3u:39]: Seems geo-blocked.

Comment (sguinetti) [reply channels/ve.m3u]: Added [Geo-blocked] tag.

**Review by BellezaEmporium — APPROVED** [review]

**Review by Dum4G — APPROVED** [review]

**Pull request merged**